platforms/core-configuration/file-collections/src/test/groovy/org/gradle/api/internal/file/collections/DefaultConfigurableFileCollectionSpec.groovy

```diff
@@ -2041,9 +2041,5 @@ class DefaultConfigurableFileCollectionSpec extends FileCollectionSpec {
         ["src1"]        | _             | ["src1"]          | "convention used when no explicit value"      | { }
         ["src3"]        | ["src3"]      | ["src1"]          | "explicit value overrides convention"         | { }
         ["src1"]        | ["src3"]      | ["src1"]          | "convention used when explicit unset"         | { it.unset() }
-        ["src1", "src2"]| _             | _                 | "from() after convention honors it"           | { it.convention("src1"); it.from("src2") }
```

Comment:
❓ Why was this removed?

Reply:
This was removed by accident; restored.
Thank you for spotting this!